cmd/src/batch_common.go

```diff
@@ -82,6 +82,8 @@ type batchExecuteFlags struct {
 
 	// EXPERIMENTAL
 	textOnly bool
+
+	sourcegraphVersion string
```

Comment:
we pass this down so we can do feature detection without needing to talk to Sourcegraph